alsdkdefs/__init__.py

```diff
@@ -228,7 +228,7 @@ def validate_operation_ids(spec):
             methods = _get_path_methods(path_obj)
             for method in methods:
                 operationid = path_obj[method].get(OpenAPIKeyWord.OPERATION_ID, None)
-                pattern = '^[a-z_]+$'
+                pattern = '^[a-z][a-z0-9_]*$'
```

Comment:
```suggestion
                pattern = '^[A-Za-z_][a-z0-9_]*$'
```

Reply:
i have nothing against op_ids being more fluent yet limited to the python function name syntax.

Reply:
@msayler any objections^^

Reply:
none from product side.

Reply:
I'm not sure starting with `_` will be acceptable everywhere / to everyone.  I don't object, I just want to make sure we don't open it up to something that will be hard to work around with our tooling.

Reply:
alright, let's narrow it down to just alpha chars.